spec/models/pt_session_spec.rb

```diff
@@ -26,12 +26,38 @@
   end
 
   describe 'self.past_week' do
-    xit 'returns pt_sessions logged in the past week' do
+    before(:each) do
+      @user = create(:user)
+    end
+    it 'returns only the logs between today and the past 7 days' do
```

Comment:
I know you don't know this yet, but after you wrote this code, I made a change to the `factories/pt_sessions.rb` file which allows the factory to create this `user` for you. 

From this:
```
factory :pt_session do
    user { nil }
...
```

To this:
```
factory :pt_session do
    user
...
```

So we won't need the `before` blocks or the `user_id: @user.id` anymore.